### PR TITLE
2543: Remove poi list horizontal scroll bar

### PR DESCRIPTION
**Pull request by lunars97**
### Short description

There is a horizontal scroll bar on Poi List Items on web. There should not be any horizontal scroll bar.

### Steps to reproduce
- Go to [https://integreat.app/muenchen/de/locations](url)
- Look at the bottom of the list panel

### Side effects
none

### Resolved issues

Fixes: #2543 

---

### CODE REVIEW

**Review by LeandraH — COMMENTED** [review]
I'm still getting a (smaller) horizontal scrollbar. This is in Firefox with the window being 1512x831.

image

**Review by ztefanie — COMMENTED** [review]
Tested on Chrome, no horizontal scroll bar anymore :tada:
But there is no hyphen displayed for me neither in Chrome, nor in Firefox. If it is a lot of effort to add it, i am fine with the current solution

**Review by LeandraH — APPROVED** [review]
Looks good to me now, and plenty of hyphens in Firefox

**Review by f1sh1918 — COMMENTED** [review]
Nice find :)
Just one small comment.

File: web/src/components/PoiListItem.tsx [hunk 1]
```diff
@@ -53,7 +53,9 @@ export const Description = styled.div`
   flex-grow: 1;
   padding: 0 22px;
   align-self: center;
-  word-wrap: break-word;
+  overflow-wrap: break-word;
```
Comment (f1sh1918) [inline web/src/components/PoiListItem.tsx:56]: don't think this is needed. Couldn't find an issue in chrome nor firefox

**Review by f1sh1918 — APPROVED** [review]

**Comment by Bekmuratovgo** [comment]
> ### Short description
> There is a horizontal scroll bar on Poi List Items on web. There should not be any horizontal scroll bar.
> 
> ### Steps to reproduce
> * Go to [https://integreat.app/muenchen/de/locations](url)
> * Look at the bottom of the list panel
> 
> ### Side effects
> none
> 
> ### Resolved issues
> Fixes: #2543

excellent work! :)

**Pull request merged**